purchase_order_type_dashboard/models/purchase_order_type.py

```diff
@@ -0,0 +1,60 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class PurchaseOrderType(models.Model):
+    _inherit = "purchase.order.type"
+
+    purchase_order_ids = fields.One2many(
+        comodel_name="purchase.order",
+        inverse_name="order_type",
+        string="Purchase Orders",
+    )
+    state_rfq_po_count = fields.Integer(string="RFQs", compute="_compute_po_counts")
+    invoice_status_no_po_count = fields.Integer(
+        string="Nothing to Bill", compute="_compute_po_counts"
+    )
+    invoice_status_ti_po_count = fields.Integer(
+        string="Waiting Bills", compute="_compute_po_counts"
+    )
+
+    @api.depends(
+        "purchase_order_ids",
+        "purchase_order_ids.state",
+        "purchase_order_ids.invoice_status",
+    )
+    def _compute_po_counts(self):
+        rfq_states = ("draft", "sent", "to approve")
+        po_states = ("purchase", "done")
+        po_model = self.env["purchase.order"]
```

Comment:
```suggestion
        PurchaseOrder = self.env["purchase.order"]
```
Not a blocker, but I think Odoo has adopt to use camel case for object declaration like this.